packages/typescript/test/ts_project/filegroup/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+load("//packages/typescript:index.bzl", "ts_project")
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["*.ts"]),
+)
+
+ts_project(
+    srcs = [":srcs"],
+    composite = True,
+    extends = "//packages/typescript/test/ts_project:tsconfig-base.json",
+    tsconfig = "tsconfig.json",
+    visibility = ["//packages/typescript/test:__subpackages__"],
+    # Use @babel/parser since the package.json is required to resolve "typings" field
+    # Repro of #2044
+    deps = [
+        "@npm//@babel/parser",
+        "@npm//@babel/types",
+    ],
+)
```

Comment:
Since c.js comes from a source in the filegroup and is therefore not predeclared as an output, so it can't be asserted by using a filegroup like in other test packages. If the ts_project rule builds, then that is a pretty good assertion, but if we want to check that `c.js` is output we would need to write a rule implementation that checks the files resolved by a sources attr. Is there an existing utility that does this?